### DISCUSSION TITLE
Welcome to toru Discussions!

### DISCUSSION THREAD

**Discussion by sweetbbak** [sweetbbak/toru#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Give feedback

**Comment by evandrojr** [comment]
Does it seed while running? If not it would be a good feature.

**Reply by sweetbbak** [reply]
It does. I want to flesh things out a bit more so that the user has more control over things like that.

**Comment by ShedowKonfas** [comment]
2025/06/16 19:29:38 error getting the nyaa page: error requesting results: Get "https://nyaa.si?c=1_2&f=0&o=desc&p=1&q=naruto&s=id": dial tcp 10.100.0.114:443: i/o timeout
Что делать??

**Reply by sweetbbak** [reply]
try using this flag `--proxy="https://nyaa.ink.iss"` you may be getting geo-blocked. I can take a closer look at it later, but things seem to be working for me.

**Reply by ShedowKonfas** [reply]
unknown flag `proxy'

**Reply by sweetbbak** [reply]
its gotta be like this:
```
toru --proxy="https://nyaa.ink" latest
```
if that genuinely doesn't work then you really gotta update the binary lol
